src/main/java/com/example/demo/Carddata/CarddataService.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.Carddata;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CarddataService {
+
+    @Autowired
+    public CarddataService() {
+
+    }
+
+    public List<Carddata> getCarddata() {return List.of();}
```

Comment:
Where is DataRepository? Will it be added later?

Reply:
done